app/models/reference/CountryCode.scala

```diff
@@ -22,15 +22,14 @@ case class CountryCode(code: String)
 
 object CountryCode {
 
-  implicit val format: Format[CountryCode] =
-    new Format[CountryCode] {
-      override def writes(o: CountryCode): JsValue = JsString(o.code)
+  implicit val countryCodeWrites = new Writes[CountryCode] {
+    override def writes(countryCode: CountryCode): JsValue = JsString(countryCode.code)
+  }
```

Comment:
Can this be converted into a single abstract method? Something like

`implicit val countryCodeWrites: Writes[CountryCode] = (countryCode: CountryCode) => JsString(countryCode.code)`